common/python/error_code/enclave_error.py

```diff
@@ -31,4 +31,5 @@ class EnclaveError(IntEnum):
     # this should be converted to ENCLAVE_ERR_SYSTEM for reporting.
     ENCLAVE_ERR_SYSTEM_BUSY = -10,
     ENCLAVE_ERR_CRYPTO = -11,
-    ENCLAVE_ERR_INVALID_WORKLOAD = -12 # Invalid workload ID
+    # Invalid workload ID
```

Comment:
Rebase and that lint issue will be gone. :)
This is also not bad though.

Reply:
rebased